src/pages/SignUp/SignUp.js

```diff
@@ -1,9 +1,341 @@
+                    개인정보를 다음의 목적을 위해 활용합니다.
+                  </p>
+                  <p>
+                    가. 서비스 제공에 관한 계약 이행 및 서비스 제공에 따른
+                    요금정산, 콘텐츠 제공, 물품(상품 또는 경품)배송 또는 청구서
+                    등 발송, 금융거래, 본인인증, 구매 및 대금결제, 요금추심 등
+                    정보안내
+                  </p>
+                  <p>나. 서비스 제공에 있어서 고객에게 맞춤화된 서비스 제공</p>
+                  <p>
+                    다. 회원관리 (회원가입을 하는 경우에 한함) 회원제 서비스
+                    이용에 따른 본인확인, 불량회원의 부정이용 방지와 비인가
+                    사용방지, 가입의사 확인, 가입 및 가입횟수 제한, 만14세 미만
+                    아동 개인정보 수집 시 법정대리인 동의여부 확인, 추후
+                    법정대리인 본인확인, 분쟁조정을 위한 기록보존, 불만처리 등
+                    민원처리, 고지사항 전달
+                  </p>
+                  <p>라. 상담요청에 대한 회신</p>
+                  <p>
+                    마. 마케팅 및 광고에 활용 신규서비스(제품) 개발 및 특화,
+                    인구통계학적 특성에 따른 서비스 제공 및 광고 게재, 접속 빈도
+                    파악, 회원의 서비스 이용에 대한 통계, 새로운 서비스 안내,
+                    신상품이나 이벤트 정보안내 등 광고성 정보 전달 이벤트,
+                    신제품 출시, 쿠폰, 세일 및 광고성 뉴스레터/SMS 발송, 소셜
+                    미디어 회원 타겟팅 광고 등 일체의 마케팅 활동에 있어 회원의
+                    성명,주소,이메일 및 휴대전화번호를 활용합니다.
+                  </p>
+                  <p>
+                    바. 마케팅 및 광고에 활용되는 개인정보 보유기간 수신동의
+                    철회 접수 및 회원탈퇴 시 까지
+                  </p>
+                  <p>3. 개인정보의 제공 및 공유 </p>
+                  <p>
+                    가. (주)동일드방레는 고객의 사전 동의가 있거나 법률에서
+                    정하는 경우 외에는 고객의 개인정보를 제3자에게 제공하거나
+                  </p>
+                  <p>
+                    2. "개인정보의 수집 및 이용목적"에서 고지한 목적 외의 용도로
+                    이용하지 않습니다.
+                  </p>
+                  <p>
+                    나. 개인정보를 제공할 경우에는 사전에 고객에게 개인정보를
+                    제공받는 제3자, 제공하는 개인정보항목명, 개인정보 제공의
+                    목적, 제공하는 개인정보가 언제까지 어떻게 보호, 관리되는지,
+                    고객이 종전에 동의한 사항, 고객의 동의하지 않을 권리와
+                    의사표시 방법, 기타 고객의 동의를 얻기 위하여 필요한 사항에
+                    대해 개별적으로 서면, 전자우편, 전화 등을 통해 고지하여
+                    동의를 구하는 절차를 거치게 되며, 고객이 동의하지 않는
+                    경우에는 제3자에게 정보를 제공하지 않습니다.
+                  </p>
+                  <p>
+                    4. 개인정보의 취급위탁 (주)동일드방레는 서비스 향상을 위해서
+                    고객의 개인정보를 외부전문업체에 위탁하여 처리할 수
+                    있습니다. 가. 개인정보의 처리를 위탁하는 경우에는 관계법령에
+                    따라 필요한 경우 그 사실을 고객에게 고지하고 동의를 받도록
+                    하겠습니다.
+                  </p>
+                  <p>
+                    나. 개인정보의 처리를 위탁하는 경우에는 위탁계약 등을 통하여
+                    위탁업무의 목적 및 범위, 개인정보에 대한 기술적∙관리적
+                    보호조치, 서비스제공자의 개인정보보호 관련 지시엄수 등
+                    서비스제공자에 대한 감독, 개인정보에 관한 비밀유지, 제3자
+                    제공 및 위탁목적 외 처리의 금지 및 사고시의 책임부담 등을
+                    명확히 규정하고 당해 계약내용을 서면 또는 전자적으로
+                    보관하겠습니다.
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div className="useragreement">
```

Comment:
```suggestion
          <div className="userAgreement">
```
다른 부분 카멜 케이스로 되어 있으니까 여기도 그렇게 하는게 좋겠습니다

Reply:
자세한 코드 리뷰 감사합니다 🙂